src/site/content/en/blog/youtube-media-capabilities/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: YouTube enhances UX with Media Capabilities API
+subhead: |
+  Ensure buttery smooth playback when your users are on different devices.
+authors:
+  - beaufortfrancois
+date: 2019-06-05
+hero: hero.jpeg
+hero_position: top
+alt: Youtube application screengrab
+description: |
+  In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase
```

Comment:
```suggestion
  Using the Media Capabilities API lets you better target video codecs, frame rates, and resolutions based on the capabilities of your users' devices.
```
This seems better aligned with the content of the article?

Reply:
Done.